.github/workflows/unified-docker.yml

```
@@ -123,14 +127,81 @@ jobs:
+      - setup
+      - build
+    if: ${{ needs.setup.outputs.matrix != '[]' && (github.event_name == 'schedule' || inputs.push_to_ghcr == true) }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        backend: ${{ fromJSON(needs.setup.outputs.matrix) }}
+
+    steps:
+      - name: Set date tag
+        run: echo "DATE_TAG=$(date -u +%Y-%m-%d)" >> $GITHUB_ENV
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:1:41: Double quote to prevent globbing and word splitting

(shellcheck)